testdata/config/prometheus.yaml

```diff
@@ -92,9 +92,8 @@ kind: rule
 metadata:
   name: promtcp
   namespace: istio-config-default
-  labels:
-    istio-protocol: tcp # needed so that mixer will only execute when context.protocol == TCP 
 spec:
+  match: context.protocol == "tcp"
```

Comment:
is our style recommendation that `context.protocol` clauses should come first?  or second, after `destination.service` clauses?   anywhere?

Reply:
Our code does not require anything specific. It would be an unnecessary restriction.
So `anywhere`

Reply:
This is a bit surprising, given that we want `destination.service` to always be the first (per previous discussions).

I get that the code doesn't require an ordering, but I would figure that we would want to recommend a "best practice" of sorts here too.

Reply:
In our examples we can choose destination.service to be first when both are present. But I don't want to state it as an additional best practice. Our config has enough real elements for users to worry about. 

Users can choose to follow our examples.